README.md

```diff
@@ -8,17 +8,16 @@
 
 Yet another **open-source** GitHub client app but unlike any other app, FastHub was built from scratch.
  
- 
-## Download
- 
- **FastHub** can be downloaded on [Google Play](https://play.google.com/store/apps/details?id=com.fastaccess.github) or [F-Droid](https://f-droid.org/repository/browse/?fdid=com.fastaccess.github):
```

Comment:
could you while at it, remove the F-Droid section at all since we dropped Fdroid support.